modules/core/client/less/layout/board.less

```diff
@@ -80,10 +86,10 @@
   .page-header {
     margin: 0;
     h1 {
-      .h1();
+      // .h1(); @TODO find out what happened to this, not defined anywhere
```

Comment:
Could be coming from Bootstrap less files.

Reply:
I had looked in bootstrap, but couldn't find it. I found all the other references to bootstrap mixins. Perhaps an older version of bootstrap did define them? Or it's just a misunderstanding (as bootstrap defines .h1, .h2 as _classes_, but not mixins).

I can't actually see anywhere that defines a `.page-header` inside a `.board`, so suspect it's not even being used, but don't want to attempt a CSS cleanup right now...